go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	github.com/bits-and-blooms/bitset v1.7.0 // indirect
 	github.com/consensys/bavard v0.1.13 // indirect
 	github.com/consensys/gnark-crypto v0.12.1 // indirect
-	github.com/cosmos/ledger-cosmos-go v0.13.2 // indirect
+	github.com/cosmos/ledger-cosmos-go v0.12.4 // indirect
```

Comment:
👍

Reply:
wait is this the ledger fix? I thought we bumped this to fix ledger support?

https://github.com/celestiaorg/celestia-app/pull/2678/files#r1402395173

Reply:
https://github.com/celestiaorg/cosmos-sdk/pull/357#discussion_r1385289378

Reply:
So either 0.12.4 or 0.13.2 can be used. We should use 0.12.4 here because that matches the cosmos-sdk.

Reply:
cool thanks